record-boot-success/record-boot-success.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Record Boot Success
+
+[Service]
+Type=dbus
+BusName=com.endlessm.RecordBootSuccess
+ExecStart=/usr/lib/eos-boot-helper/record-boot-success/record-boot-success
```

Comment:
`$libexecdir` here again.

Reply:
Same as for `com.endlessm.RecordBootSuccess.service`.